### ISSUE TITLE
Read images for generate_tfrecord.py

### ISSUE THREAD

**Issue by alexevag** [kylekaplan/Object_Detection-Logos#1]
I cannot understand how do you read data in line 151 in generate_tfrecord.py
path = os.path.join(os.getcwd(), 'object_detection', 'Logos', 'images')

can you explain it??
do you have a folder with all images??

**Comment by kylekaplan** [comment]
Yes, if you look here https://github.com/kylekaplan/Object_Detection-Logos/tree/master/models/research/object_detection/Logos/images you will see folders "flickr_logos_27_dataset_images" and "qmul_toplogo10" the images are omitted because it's too much to commit to github but you can download the images online by simply googling those folder names
